pykechain/models/property2_selectlist.py

```diff
@@ -63,7 +79,7 @@ def options(self):
         List of options of this property to select from.
 
         """
-        return self._value_choices
+        return copy(self._value_choices)
```

Comment:
I am not sure if this is the right `copy` method. I am always in doubt. If you are sure it is correct, then no worries. Just double check

Reply:
`copy` copies values, not object references. To be on the safe side, i'll replace it with `deepcopy`, although for a list of strings that is a bit overkill :)